### PR TITLE
docs: release note update and system requirements

### PR DESCRIPTION
**Pull request by marcobonita**

### CODE REVIEW

**Review by benjaminParisel — APPROVED** [review]

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -10,6 +10,20 @@ The {bonitaVersion} version is in development.
 
 We are excited to announce the release of xref:applications:bonita-ui-builder.adoc[Bonita UI Builder], our new and enhanced development environment for creating all kind of interfaces, pages and forms for your applications! It brings along a host of advanced features and improved usability, designed to help you create compelling, responsive, and user-friendly interfaces.
 
+Key benefits:
+
+* Rapidly create and deploy UIs to reduce time to market
+* Connect UIs with Bonitasoft for end-to-end alignment
+* Tailor UIs to specific needs with advanced options
+* A drag-and-drop interface empowers all users, regardless of technical skill
+* Built with React for dynamic, responsive design
```
Comment (jeromecambon) [inline modules/ROOT/pages/release-notes.adoc:19]: Not sure we need to mention React, which is fully hidden to the users. One can have responsive design without React

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -10,6 +10,20 @@ The {bonitaVersion} version is in development.
...
+
+Features include:
+
+* Drag & drop tools, 45+ widgets, React interfaces
```
Comment (jeromecambon) [inline modules/ROOT/pages/release-notes.adoc:23]: Same comment about React

File: modules/applications/pages/ui-builder/faq.adoc [hunk 3]
```diff
@@ -18,6 +18,10 @@ Bonita UI Builder is available by default and for no additional cost in all http
 Please refer to the xref:initiation-manual.adoc[initiation manual]. If you have any other additional questions, reach out to our https://csc.bonitacloud.bonitasoft.com[Support team].
 
 
+=== What are the system requirements? 
+Your system/server should have the following configuration for optimal performance: 2 virtual CPU and 4 GB of memory
```
Comment (jeromecambon) [inline modules/applications/pages/ui-builder/faq.adoc:22]: ... should have the **minimal** following configuration...

**Review by jeromecambon — COMMENTED** [review]
Few comments

**Pull request merged**